listenbrainz/db/user_timeline_event.py

```diff
@@ -79,6 +79,24 @@ def create_user_notification_event(user_id: int, metadata: NotificationMetadata)
         metadata=metadata
     )
 
+def delete_user_recommendation_notification_event(
+        id: int,
+        user_id:int
+) -> bool:
+    ''' Deletes recommendation and notification event using id'''
+    try:
+        with db.engine.connect() as connection:
+            result = connection.execute(sqlalchemy.text('''
+                    DELETE FROM user_timeline_event
+                    WHERE user_id = :user_id
```

Comment:
Is user_id even required for this query?

Reply:
To make sure we don't let a user delete another user's events. we use the same the technique for pinned recs.
https://github.com/metabrainz/listenbrainz-server/blob/29baea454b05677aed9f5909085cec3ce0c56db7/listenbrainz/db/pinned_recording.py#L82-L91